jenkins-pipelines/cdcReplicationPipeline.jenkinsfile

```diff
@@ -66,6 +74,10 @@ pipeline {
                 SCT_INSTANCE_PROVISION="${params.provision_type}"
                 SCT_IP_SSH_CONNECTIONS="${params.ip_ssh_connections}"
                 SCT_STORE_RESULTS_IN_ELASTICSEARCH="false"
+
+                // I need to include "SCT_" inside the variable's name so hydra.sh passes it to the test.
+                // However, I can't put "SCT_" at the beggining or SCT's config validator will reject the variable...
+                STRESSOR_SCT_="${params.stressor}"
```

Comment:
Better to use SCT_ prefix. in this case all parameters automatically appeared in ClusterTester.params dict